packages/datadog_flutter_plugin/lib/datadog_flutter_plugin_web.dart

```diff
@@ -10,34 +10,41 @@ import 'package:flutter_web_plugins/flutter_web_plugins.dart';
 
 import 'src/datadog_configuration.dart';
 import 'src/datadog_sdk_platform_interface.dart';
+import 'src/logs/ddlogs_platform_interface.dart';
+import 'src/logs/ddlogs_web.dart';
+import 'src/rum/ddrum_platform_interface.dart';
+import 'src/rum/ddrum_web.dart';
 
 /// A web implementation of the DatadogSdk plugin.
 class DatadogSdkWeb extends DatadogSdkPlatform {
   static void registerWith(Registrar registrar) {
     DatadogSdkPlatform.instance = DatadogSdkWeb();
 
-    // TODO: Replace platforms across all plugins
+    DdLogsPlatform.instance = DdLogsWeb();
+    DdRumPlatform.instance = DdRumWeb();
   }
 
   @override
-  Future<void> setSdkVerbosity(Verbosity verbosity) {
-    return Future.value();
-  }
+  Future<void> setSdkVerbosity(Verbosity verbosity) async {}
 
   @override
-  Future<void> setTrackingConsent(TrackingConsent trackingConsent) {
-    return Future.value();
-  }
+  Future<void> setTrackingConsent(TrackingConsent trackingConsent) async {}
 
   @override
-  Future<void> setUserInfo(
-      String? id, String? name, String? email, Map<String, dynamic> extraInfo) {
-    return Future.value();
-  }
+  Future<void> setUserInfo(String? id, String? name, String? email,
+      Map<String, dynamic> extraInfo) async {}
```

Comment:
What about these methods? Is it that browser SDK doesn't define such interface and these cannot be forwarded to `*Platform.instance` or is it no-op for some other reason?

Reply:
Actually, it's probably a miss. I don't think I have an integration test fro setUserInfo and I probably should....

Reply:
I'll add in a separate PR though.